devito/compiler.py

```diff
@@ -287,3 +295,14 @@ def jit_compile_and_load(ccode, basename, compiler=GNUCompiler):
     jit_compile(ccode, basename, compiler=compiler)
 
     return load(basename, compiler=compiler)
+
+
+def jit_compile_only(ccode, basename, compiler=GNUCompiler):
```

Comment:
I don't really see the point of this function - it simply invokes `jit_compile` and returns one of the input arguments. Can you not just use `jit_compile` instead?